mon-pix/app/models/campaign-participation-overview.js

```diff
@@ -6,18 +6,18 @@ export default class CampaignParticipationOverviews extends Model {
   @attr('boolean') isShared;
   @attr('date') sharedAt;
   @attr('string') organizationName;
-  @attr('string') assessmentState;
+  @attr('string') status;
   @attr('string') campaignCode;
   @attr('string') campaignTitle;
   @attr('date') campaignArchivedAt;
   @attr('number') masteryRate;
   @attr('number') totalStagesCount;
   @attr('number') validatedStagesCount;
 
-  get status() {
+  get cardStatus() {
```

Comment:
Le nommage fait penser au statut de la carte.
Je suggère:

```suggestion
  get campaignStatus() {
```

Reply:
cela ne correspond pas au statut de la campagne. `cardStatus` était plus proche de la vérité selon moi